INSTALLATION.md

```diff
@@ -58,19 +58,24 @@ If you have downloaded binaries you can just jump to section
 NZBGet is developed on a linux-system, but it runs on other
 POSIX platforms.
 
-NZBGet absolutely needs the following libraries:
+To build NZBGet you will need:
 
- - libstdc++     (usually part of compiler)
+For configuring and building:
+ - [CMake](https://cmake.org/)
+ - [GCC](https://gcc.gnu.org/)
+
+   or
+ - [CLang](https://clang.llvm.org/)
+
+Libraries:
  - [libxml2](https://gitlab.gnome.org/GNOME/libxml2/-/wikis/home)
  - [Boost.JSON](https://www.boost.org/doc/libs/1_84_0/libs/json/doc/html/index.html)
- - [Boost.Optional](https://www.boost.org/doc/libs/1_84_0/libs/optional/doc/html/index.html)
+> If you can't install Boost.Json on your system, jsut skip it. CMake will take care of it.
```

Comment:
just

Reply:
Better: If you face issues with Boost.Json on your system, you can skip it - CMake will take care of it.